apps/a_dndtoggle/a_dndtoggle.app.js

```diff
@@ -0,0 +1,44 @@
+
+const modeNames = [/*LANG*/"Noisy", /*LANG*/"Alarms", /*LANG*/"Silent"];
+let bSettings = require('Storage').readJSON('setting.json',true)||{};
+let current = 0|bSettings.quiet;
+//0 off
+//1 alarms
+//2 silent
+
+console.log("old: " + current);
+
+switch (current) {
+	case 0:
+		bSettings.quiet = 2;
+		Bangle.buzz();
+		setTimeout('Bangle.buzz();',500);
+		break;
+	case 1:
+		bSettings.quiet = 0;  
+		Bangle.buzz();
+		break;
+	case 2:
+		bSettings.quiet = 0;
+		Bangle.buzz();
+		break;
+	default:
+		bSettings.quiet = 0;
+		Bangle.buzz(); 
+}
```

Comment:
The last three cases are all the same, and we always buzz, so you could just replace it with
```js
if (current === 0) {
    bSettings.quiet = 2;
    setTimeout('Bangle.buzz();',500);
} else {
    bSettings.quiet = 0;
}
Bangle.buzz(); 
```

Reply:
This is for future use. I will make the behavior configurable.